x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -251,6 +253,10 @@ public final void test() throws Throwable {
                 "enrich can't load fields in csv tests",
                 testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.ENRICH_LOAD.capabilityName())
             );
+            assumeFalse(
+                "can't use rereank in csv tests",
+                testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.ENRICH_LOAD.capabilityName())
```

Comment:
should this use `EsqlCapabilities.Cap.RERANK`?

Reply:
Good catch. Fixed. Thank you!!!